### PR TITLE
Pies/rootwiches

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
This PR has a few simple aims; 2 new rootbread sandwiches, slicable versions of most pies and a new rootbread crafting method.
Sliceable pies added:
-Cherry
-Apple
-Frosty
-Meat
-Tofu
-Xenomeat

Sammiches: Peanut butter & jelly and peanutbutter & banana.

image
image
## Why It's Good For The Game
Lizard food has been around for a while; if it's ever made it's usually the same handful of easy dishes repeated and recycled. Why can't a lizard enjoy a good sandwich? Even the most subhuman crewmembers deserve a treat every once in a while!
Sliceable pies are good; because watching someone eat an ENTIRE pie to themselves is insulting and justifies the chef leaping over the counter and feeding them their own entrails!

EDIT: I forgot to mention in my first pass why the rootdough alternative was good; I noticed that in cakebatter you are able to replace eggs with soymilk, to allow an alternate way of making it. I repeated this trend of replacing eggs with soymilk as it's a replacement for eggs and is also lizard safe.
## Changelog
:cl:
add: Most pies can now be sliced rather than being consumed whole.
add: Rootdough can be crafted using soy milk in place of eggs.
add: Two new lizard-safe rootbread sandwiches can be crafted.
/:cl:

### CODE REVIEW

**Review by Wallemations — APPROVED** [review]
Mmm pie

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm [hunk 1]
```diff
@@ -187,6 +187,17 @@
...
+	name = "Rootdough"
+	reqs = list(
+		/obj/item/food/grown/potato = 2,
+		/datum/reagent/consumable/soymilk = 15,
+		/datum/reagent/consumable/korta_flour = 5,
+		/datum/reagent/water = 10
+	)
+	result = /obj/item/food/rootdough
+	category = CAT_LIZARD
+
```
Comment (Jacquerel) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: This change is not noted either in the PR body or change notes, and resultingly also isn't justified by any of the explanatory text. Did you include it by mistake?

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm (hunk 1 above)
Comment (xXPawnStarrXx) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: It's listed in there, it's also listed in the screenshot.

Comment (xXPawnStarrXx) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: "This PR has a few simple aims; 2 new rootbread sandwiches, slicable versions of most pies and a **new rootbread crafting method**."

Comment (xXPawnStarrXx) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: I will add it to the changelog shortly when I return to my PC.

Comment (Jacquerel) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: Please also add why it is good for the game.

Comment (xXPawnStarrXx) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm:200]: Done.

**Pull request merged**